MAUI/CheckBox/Getting-Started.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: post
+title: Getting Started with .NET MAUI CheckBox | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI CheckBox (SfCheckBox) control, its elements and more.
+platform: Maui
+control: SfCheckBox
+documentation: ug
+---
+
+# Getting Started with .NET MAUI CheckBox
+
+This section explains the steps required to work with the `.NET MAUI CheckBox` control.
+
+## Creating an application with .NET MAUI CheckBox
+
+This section explains the steps required to work with the `CheckBox` control for .NET MAUI.
+
+## Adding a .NET MAUI CheckBox reference
+
+Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add `.NET MAUI CheckBox` to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.Buttons`, and install it.
```

Comment:
Syncfusion .NET MAUI controls are available on [Nuget.org](https://www.nuget.org/). To add the `.NET MAUI CheckBox` to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.Buttons`, and install it.